src/extra.css

```diff
@@ -130,7 +130,7 @@ a:hover, a:active {
   font-size: 1.2em;
 }
 .home-action {
-  width: 45%;
+  width: calc(50% - 2em);
```

Comment:
What about browser than does not support `calc()`?

Reply:
All except Opera Mini according to https://caniuse.com/#feat=calc

Reply:
o_O wow, I thought it was less supported.